addons/website/views/snippets/s_popup.xml

```diff
@@ -57,6 +58,7 @@
                 <we-button data-select-data-attribute="mouseExit">On Exit</we-button>
             </we-select>
             <we-input class="o_we_inline" string="Delay" data-select-data-attribute="" data-attribute-name="showAfter" data-unit="s" data-save-unit="ms" data-dependencies="show_delay"/>
+            <we-input class="o_we_inline" string="Period between consents" data-select-data-attribute="7days" data-attribute-name="consentsDelay" data-unit="days"/>
```

Comment:
Don't use data-unit for this, add days in a span next the the we-input I guess. Otherwise it won't be translated.

Reply:
Fixed another way, I prefer to keep the DOM as it is supposed to be (we style).